boards/riscv/hpm6750evkmini/doc/index.rst

```diff
@@ -0,0 +1,109 @@
+.. _hpm6750evkmini:
+
+HPMicro HPM6750EVKMINI
+######################
+
+Overview
+********
+
+The HPM6750 is a dual-core flashless MCU running 816Mhz.
+It has a 2MB continuous on-chip ram.
+Also, it provides various memory interfaces, including SDRAM, Quad SPI NOR Flash, SD/eMMC.
+
+The figure shows the HPM6750EVKMINI board.
+
+.. image:: img/hpm6750evkmini.png
+     :align: center
+     :alt: HPM6750EVKMINI
+
+Hardware
+********
+
+The HPM6750EVKMINI platform integrates 2 cores 32-bit 816MHz RISC-V CPUs, DSP,
+2M RAM, Cache, SPI flash memory, ethernet controller and other peripherals.
+
+- HPM6750IVM MCU (816Mhz, 2MB OCRAM)
+- Onboard Memory
+  - 128Mb SDRAM
+  - 64Mb Quad SPI NOR Flash
+- Display & Camera
+  - LCD connector
+  - Camera (DVP)
+- WiFi
+  - RW007 over SPI
+- USB
+  - USB type C (USB 2.0 OTG) connector x2
+- Audio
+  - Mic
+  - DAO
+- Others
+  - TF Slot
+  - FT2232
+  - Beeper
+  - RGB LED
+- Expansion port
+  - ART-PI extension port
+
+Serial Port
+===========
+
+The HPM6750EVKMINI platform has 2 UARTs.
+The Zephyr console output is by default assigned to UART0 and the default
+settings are 115200 8N1.
+
+Programming and debugging
+*************************
+
+Building
+========
+
+You can build applications in the usual way. Here is an example for
+the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :board: hpm6750evkmini
+   :goals: build
+
+Flashing
+========
+
+ ``CONFIG_XIP=n``, you can load the program (``zephyr.elf``) into RAM directly and execute it.
```

Comment:
`CONFIG_XIP=n` is the default setting correct?  Can you clearly state that here?  As is, this implies the user should override this setting before building.

Reply:
This version does not yet enable xpi flash, so it can only run code in ram at the moment.

Reply:
If XIP flash is not supported, please state that in the documentation.
For example:
```
Execute-in place flash is not support (CONFIG_XIP-=n), so the program (zephyr.elf) is loaded into and executed directly from RAM.
```